ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml

```diff
@@ -81,7 +81,7 @@ Item {
                     parent.font.underline = false
                 }
                 onClicked: {
-                    inviteFriendsPopup.open();
+                    Global.openPopup(inviteFriendsPopup)
```

Comment:
if used via Global.openPopup, maybe it's good moment to move `inviteFriendsPopup` component to `Popups`?

Reply:
It's used only from here afaics

Reply:
Yeah, can do

Reply:
On a second thought, I decided not to... I'd rather avoid these refactors in a simple bugfix PR like this